<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HTableData</class>
 <widget class="QDialog" name="HTableData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="searchButton">
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="exportButton">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="showDepth">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Level</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="showHierarchy">
     <property name="text">
      <string>Show Hierarchy</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
